digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2/linearasset/AssetFiller.scala

```diff
@@ -489,22 +489,30 @@ class AssetFiller {
     }
   }
 
-  def fillRoadLinksWithoutAsset(roadLinks: Seq[RoadLink], linearAssets: Map[String, Seq[PersistedLinearAsset]], typeId: Int, changedSet: Option[ChangeSet] = None): Seq[PieceWiseLinearAsset] = {
-    val fillOperations: Seq[(RoadLink, Seq[PersistedLinearAsset]) => Seq[PersistedLinearAsset]] = Seq(
+  def adjustAssets(roadLinks: Seq[RoadLink], changeSet: ChangeSet, linearAssets: Map[String, Seq[PersistedLinearAsset]], typeId: Int): (Seq[PersistedLinearAsset], ChangeSet) = {
```

Comment:
Tämä adjustAssets ei taida erota muuten fillTopology:sta kuin tehtävien operaatioiden osalta? Jos näin, niin periaatteessa voisi lisätä fillTopologyyn parametrin jonka perusteella sitten päätetään mitkä operaatiot ajetaan. Vähentäisi duplikaattikoodia

Reply:
Joo, loppujen lopuksi tästä tulikin käytännössä sama funktio kuin fillTopology, melkein selkein kun yhdistää nämä ja parametrisoi operaatiolistan